### DISCUSSION TITLE
Issue Running Ministark Library on Ubuntu OS - Compatibility with Linux?

### DISCUSSION THREAD

**Discussion by upendra-eth** [andrewmilson/ministark#17] [Q&A]
Hello community,

I hope this message finds you well. I've been trying to run the Ministark library on Ubuntu OS, but I've encountered an error during the process. I'm wondering if the Ministark library is intended to be compatible with Linux systems like Ubuntu, or if there might be certain adjustments needed.

The error I'm facing is [ unresolved import `ministark_gpu::metal` ]. I've followed the installation steps mentioned in the documentation carefully, and everything seemed to go smoothly up until this point.

Could someone please shed some light on whether the Ministark library is expected to work on Linux systems? If so, are there any specific prerequisites or configurations that I should be aware of? If not, are there any known workarounds or alternative libraries that I could explore?

I appreciate any insights or guidance you can provide. Thank you in advance for your assistance!

**Comment by andrewmilson** [comment]
Hey Upendra. miniSTARK is expected to work on Lunux systems. What command are you using to run miniSTARK? I suspect the "gpu" flag is enabled which is only supported on macOS.
